utils/log.go

```diff
@@ -0,0 +1,96 @@
+package utils
+
+import "fmt"
+
+func FormatWritesLog(codes []byte, line *string) {
```

Comment:
Is there any particular reason why you don't just return the string instead of this indirection?

Reply:
Because in many cases, the commands entered by the user cannot be clearly displayed using strings, such as moving the cursor forward, using the previous command, etc. Therefore, I need to display them completely. When auditing, we can clearly know that the user the behavior of.

Reply:
For example, we can log like:

```
2022/12/11 17:43:00 [write-log] {"arg":["without-istio","sleep-7b6d569576-8nz7t","sleep","21001713"]} pwdCR
2022/12/11 17:43:03 [write-log] {"arg":["without-istio","sleep-7b6d569576-8nz7t","sleep","21001713"]} lsCR
2022/12/11 17:43:15 [write-log] {"arg":["without-istio","sleep-7b6d569576-8nz7t","sleep","21001713"]} rmSPACE-hCR
2022/12/11 17:43:18 [write-log] {"arg":["without-istio","sleep-7b6d569576-8nz7t","sleep","21001713"]} UPCR
```

`UPCR` means user re-exec `rm -h` command.